lib/main.coffee

```diff
@@ -1,114 +1,80 @@
-{CompositeDisposable} = require 'atom'
+CSON = require 'season'
 path = require 'path'
-promiseHelper = require './promise-helper'
+
+grammarRE = /\/language-[^\/]+\/grammars\/[^\/]+\.(?:c|j)son/
```

Comment:
I prefer keep the configuration: because I don't want to reload all languages grammars in my workspace, but just one.
But it's possible to find way to do your needs and my needs.

Reply:
If you save a grammar `.cson` or `.json`, all affected editors will be refreshed. Only the changed grammar is reloaded. So I don't understand your complaint here.

Reply:
I want to choose which grammar I reload.
I have some grammar who don't this, because the reload system is integrated to the grammar.

Reply:
Okay, I will add a whitelist config, but if empty, all grammars are reloaded. Does that sound good?

Reply:
Perhaps a blacklist config is more appropriate.

Reply:
Yes a blacklist seems good for me.